spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -92,5 +92,21 @@
         expect(response).to have_http_status(:unauthorized)
       end
     end
+
+    context 'when the user has 9 targets' do
+      let(:targets) { create_list(:target, 9, user: user) }
+      it 'the target create correct' do
+        subject
+        expect(response).to have_http_status(:success)
+      end
+    end
+
+    context 'when the user has 10 or more targets' do
+      let!(:targets) { create_list(:target, 10, user: user) }
+      it 'the target does not create' do
+        subject
+        expect(response).to have_http_status(:bad_request)
```

Comment:
you can also check the error here.. expect that the response have something like `exceeded number of targets limit` to validate that the request is bad for this error